### PR TITLE
Migrate component to version 12.0

### PR DESCRIPTION
**Pull request by guewen**

### CODE REVIEW

File: component/__manifest__.py [hunk 1]
```diff
@@ -15,5 +17,6 @@
      'python': ['cachetools'],
  },
  'data': [],
- 'installable': False,
+ 'installable': True,
```
Comment (lmignon) [inline component/__manifest__.py:19]: @guewen It should be nice to also have the development status key into the manifest
see https://odoo-community.org/page/development-status

**Review by lmignon — APPROVED** [review]
Thank you @guewen A little comment...

File: component/__manifest__.py (hunk 1 above)
Comment (guewen) [reply component/__manifest__.py:19]: Right, I was not sure at which moment it should be put, is stable the correct status?

Comment (lmignon) [reply component/__manifest__.py:19]: IMO yes it's stable!

**Comment by guewen** [comment]
Travis is red only due to the "docs" build which cannot build without the "connector" addon

File: component/__manifest__.py [hunk 2]
```diff
@@ -3,7 +3,9 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
 
 {'name': 'Components',
- 'version': '11.0.1.1.0',
+ 'summary': 'Add capabilities to register and use decoupled components,'
+            ' as an alternative to model classes',
+ 'version': '12.0.1.0.0',
  'author': 'Camptocamp,'
            'Odoo Community Association (OCA)',
  'website': 'https://www.camptocamp.com',
```
Comment (simahawk) [inline component/__manifest__.py]: https://github.com/OCA/connector

File: component/__manifest__.py [hunk 3]
```diff
@@ -15,5 +17,7 @@
      'python': ['cachetools'],
  },
  'data': [],
```
Comment (simahawk) [inline component/__manifest__.py]: drop data

**Review by simahawk — APPROVED** [review]
minor glitches

File: component/__manifest__.py (hunk 2 above)
Comment (guewen) [reply component/__manifest__.py]: done

File: component/__manifest__.py (hunk 3 above)
Comment (guewen) [reply component/__manifest__.py]: done

**Review by lmignon — APPROVED** [review]
Once again, thank you @guewen for this migration. IMO we don't need more to officially release this addon for Odoo 12.

**Review by rvalyi — COMMENTED** [review]
I approve. Travis is green minus the doc task due to the missing connector module in branch 12.0 at this stage, so okay. Also there is very little dependency on Odoo code so it's normal there is no change needed in the code. Thanks.

**Review by rvalyi — APPROVED** [review]

**Pull request merged**